ui/imports/shared/popups/keycard/states/EnterSeedPhrase.qml

```diff
@@ -0,0 +1,328 @@
+            const words = clipboardText.split(/[, \s]+/)
+
+            let index = d.tabs.indexOf(words.length)
+            if (index === -1) {
+                return false
+            }
+
+            let timeout = 0
+            if (switchTabBar.currentIndex !== index) {
+                switchTabBar.currentIndex = index
+                // Set the teimeout to 100 so the grid has time to generate the new items
+                timeout = 100
+            }
+
+            d.mnemonicInput = []
+            timer.setTimeout(() => {
+                                 // Populate mnemonicInput
+                                 for (let i = 0; i <  words.length; i++) {
+                                     grid.addWord(i + 1, words[i], true)
+                                 }
+                                 // Populate grid
+                                 for (let j = 0; j <  grid.count; j++) {
+                                     const item = grid.itemAtIndex(j)
+                                     if (!item || !item.leftComponentText) {
+                                         // The grid has gaps in it and also sometimes doesn't return the item correctly when offscreen
+                                         // in those cases, we just add the word in the array but not in the grid.
+                                         // The button will still work and import correctly. The Grid itself will be partly empty, but offscreen
+                                         // With the re-design of the grid, this should be fixed
+                                         continue
+                                     }
+                                     const pos = item.mnemonicIndex
+                                     item.setWord(words[pos - 1])
+                                 }
+                                 d.checkMnemonicLength()
+                             }, timeout)
+            return true
+        }
+    }
+
+    ColumnLayout {
+        anchors.fill: parent
+        anchors.topMargin: Style.current.xlPadding
+        anchors.bottomMargin: Style.current.halfPadding
+        anchors.leftMargin: Style.current.xlPadding
+        anchors.rightMargin: Style.current.xlPadding
+        spacing: Style.current.padding
+        clip: true
+
+        StatusBaseText {
+            id: title
+            Layout.preferredHeight: Constants.keycard.general.titleHeight
+            Layout.alignment: Qt.AlignHCenter
+            text: qsTr("Enter key pair seed phrase")
+            font.pixelSize: Constants.keycard.general.fontSize1
+            font.weight: Font.Bold
+            color: Theme.palette.directColor1
+        }
+
+        Timer {
+            id: timer
+        }
+
+        StatusSwitchTabBar {
+            id: switchTabBar
+            Layout.alignment: Qt.AlignHCenter
+            Repeater {
+                model: d.tabs
+                StatusSwitchTabButton {
+                    text: qsTr("%1 words").arg(modelData)
```

Comment:
Plural missing, see https://hackmd.io/@status-desktop/B1eOaf-nd/%2FjNzZnH8ZRiynrrXJ--XpmQ#Plurals

Reply:
It's enter seed phrase component, won't be ever a single word here, (`tabs` contains 12, 18, 24).

Reply:
That is not the point of whole plural handling, in some languages, "words" will take a different form if it's e.g. "12 words" and "18 words"